js/views/audioOnlyScreen.js

```diff
@@ -0,0 +1,83 @@
+/*********************************************************************
+  AUDIO ONLY SCREEN
+*********************************************************************/
+const React = require('React');
+const ControlBar = require('../components/controlBar');
+const ClassNames = require('classnames');
+
+const ScrubberBar = require('../components/scrubberBar');
+
+/**
+ * The screen that is shown while the player state is playing, paused, or ended for the audio only player.
+ * Will display a title with description, a control bar, and a scrubber bar.
+ */
+class AudioOnlyScreen extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      controlBarVisible: true,
+      animate: false
+    };
+  }
+
+  render() {
+    var titleStyle = {
```

Comment:
Can we use `const` for these declarations?

Reply:
Yep